packages/next/README.md

```diff
@@ -0,0 +1,30 @@
+![Next.js Runtime](https://github.com/netlify/next-runtime/raw/main/next-on-netlify.png)
+
+<p align="center">
+  <a aria-label="npm version" href="https://www.npmjs.com/package/@netlify/next">
+    <img alt="" src="https://img.shields.io/npm/v/@netlify/next">
+  </a>
+  <a aria-label="MIT License" href="https://img.shields.io/npm/l/@netlify/next">
+    <img alt="" src="https://img.shields.io/npm/l/@netlify/next">
+  </a>
+</p>
+
+# `@netlify/next`
+
+The `@netlify/next` package makes Next.js even better.
+
+## Next.js Advanced Middleware
+
+Regular Next.js Middleware doesn’t provide access to the actual response, or allow you to modify the request. The
```

Comment:
Change to "It gives full access to the request and response objects"

Reply:
Sorry this tagged the wrong line, but it's about adding "to the" in that sentence